addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -148,6 +148,12 @@
         itemscope="itemscope" itemprop="offers" itemtype="http://schema.org/AggregateOffer">
         <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
         <div id="o_wevent_tickets" class="o_wevent_theme_bg_base shadow-sm o_wevent_js_ticket_details" data-folded-by-default="0">
+            <div t-if="failed_state == 'failed-seats'" class="alert alert-danger" role="alert">
+                <p class="mb-0">
+                    Registration failed! The number of tickets chosen is not available anymore.
+                    Someone has registered to this event a few seconds before you.
```

Comment:
Would not mention that "someone has ..." and just keep a generic "These tickets are not available anymore".